src/app/friends-list/friends-list.component.html

```diff
@@ -0,0 +1,85 @@
+<main class="main">
```

Comment:
In the end, this component becomes really huge, maybe there are some opportunities to split it into separate components?